### PR TITLE
Added ability to change order status

### LINKED ISSUE #298: Create logic for changing admin order status

### PR DESCRIPTION
**Pull request by heshvab**
- Added ability for admin to change order status

### CODE REVIEW

**Comment by D1nviz** [comment]
Make text-align center
image

File: code/src/containers/tables/orders-table/components/orders-table-body/OrdersTableBody.scss [hunk 1]
```diff
@@ -12,4 +12,23 @@
...
+}
+
+.status-select {
+  padding: 0 !important;
+
+  * {
+    border: none !important;
+    outline: none !important;
+    box-shadow: none !important;
+  }
```
Comment (mhevyk) [inline code/src/containers/tables/orders-table/components/orders-table-body/OrdersTableBody.scss]: Why there is `!important` everythere?

File: code/src/containers/tables/orders-table/components/orders-table-body/OrdersTableBody.tsx [hunk 2]
```diff
@@ -28,26 +31,61 @@ const OrdersTableBody = ({ order }: OrderTableBodyProps) => {
...
-    <AppTypography
-      className="spa-order-table__body-status"
-      variant="caption"
-      translationKey={orderItemStatus}
-    />
+  const statusBlock = (
+    <AppSelect
+      defaultValue={orderStatus}
+      value={orderStatus}
+      IconComponent={() => null}
```
Comment (mhevyk) [inline code/src/containers/tables/orders-table/components/orders-table-body/OrdersTableBody.tsx:40]: Is `() => null` required here?

File: code/src/containers/tables/orders-table/components/orders-table-body/OrdersTableBody.scss (hunk 1 above)
Comment (heorhiiss) [reply code/src/containers/tables/orders-table/components/orders-table-body/OrdersTableBody.scss]: Thanks, I used it to overwrite mui. I added comments and removed some of redundant `!important`

File: code/src/containers/tables/orders-table/components/orders-table-body/OrdersTableBody.tsx (hunk 2 above)
Comment (heorhiiss) [reply code/src/containers/tables/orders-table/components/orders-table-body/OrdersTableBody.tsx:40]: I it is needed to set icon to null. Just `null` doesn't work with ts. Not sure if there is other way

**Comment by heshvab** [comment]
> Make text-align center image

Thank you, centered

**Review by mhevyk — APPROVED** [review]

**Review by OliaHrytsak — APPROVED** [review]

**Review by D1nviz — APPROVED** [review]

**Review by dmtrth25 — APPROVED** [review]

**Pull request merged**